modules/photon-cdn/jetpack-manifest.php

```diff
@@ -1,4 +1,4 @@
-<?php 
```

Comment:
This file is automatically updated as part of the release process, so you can ignore it, there is no need to update it here.

Reply:
Thank you for the confirmation. I have now reverted the changes